BioSQL/BioSeqDatabase.py

```diff
@@ -403,6 +434,13 @@ def fetch_seqid_by_display_id(self, dbid, name):
         return rv[0][0]
 
     def fetch_seqid_by_accession(self, dbid, name):
+        """Return the internal id for a sequence using its accession.
+
+        Argunments:
+            - dbid - the internal id for the sub-database
+            - name - the accession of the sequence. Corresponds to the
+            accession column of the bioentry table of the SQL schema
```

Comment:
Please check this is valid RST, I suspect the final line should be two spaces more indented as a continuation of the bullet point.